src/client/components/map/ProjectDistrictsMap.tsx

```diff
@@ -35,13 +33,21 @@ const ProjectDistrictsMap = ({
 }) => {
   const mapRef = useRef<HTMLDivElement>(null);
   const [mapLoaded, setMapLoaded] = useState<boolean>(false);
+  const [bounds, setBounds] = useState<[number, number, number, number] | null>(null);
+  project.regionConfig &&
+    !bounds &&
+    fetchMemoizedStateBbox(project.regionConfig).then((bboxData: IStaticMetadata["bbox"]) => {
+      // Conversion from readonly -> mutable to match Mapbox interface
+      const [b0, b1, b2, b3] = bboxData;
+      setBounds([b0, b1, b2, b3]);
```

Comment:
This would work as well to go from read-only -> mutable, and is less brittle (it doesn't need to know what the structure of `bboxData` is:
```suggestion
      setBounds([...bboxData]);
```